.github/workflows/updates-e2e-basic-eas.yml

```diff
@@ -0,0 +1,119 @@
+name: Updates e2e basic EAS
+
+on:
+  workflow_dispatch: {}
+  pull_request:
+    paths:
+      - .github/workflows/updates-e2e.yml
+      - packages/expo-json-utils/**
+      - packages/expo-manifests/**
+      - packages/expo-modules-core/**
+      - packages/expo-structured-headers/**
+      - packages/expo-updates-interface/**
+      - packages/expo-updates/**
+      - packages/expo/android/**
+      - packages/expo/ios/**
+      - templates/expo-template-bare-minimum/**
+  push:
+    branches: [main, 'sdk-*']
+    paths:
+      - .github/workflows/updates-e2e.yml
+      - packages/expo-json-utils/**
+      - packages/expo-manifests/**
+      - packages/expo-modules-core/**
+      - packages/expo-structured-headers/**
+      - packages/expo-updates-interface/**
+      - packages/expo-updates/**
+      - packages/expo/android/**
+      - packages/expo/ios/**
+      - templates/expo-template-bare-minimum/**
```

Comment:
any particular reasons to separate updates-e2e-assets-eas and updates-e2e-basic-eas jobs? it looks like their trigger paths are exactly the same, so in most of cases, these two ci jobs will be triggered simultaneously.

Reply:
They build different resources into the app, so they do require separate EAS runs to build and test the app.

Reply:
We could in principle have the workflows further divided by platform and have four workflows instead of two 😀

Reply:
so we may use a [matrix](https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs) rather than a duplicated file here.

i'm still thinking of separating "basic" and "assets" as different EAS jobs have little waste on EAS workers' resources. we lose the benefit from the incremental build and rebuild react-native code between these two test cases. not sure whether we could use `eas-build-on-success` to continuously build other test cases 🤔

Reply:
Ok now we have one workflow for iOS and one for Android. Each workflow kicks off a single EAS job that builds the basic test app. Then the on-success handler runs the basic tests, deletes all the files and unpacks an archive containing the assets test app, builds that app, and then runs the assets tests. This lets us do everything in 2 EAS jobs instead of 4.